WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -397,6 +402,7 @@ open class ReaderDetailViewController: UIViewController, UIViewControllerRestora
         configureRichText()
         configureDiscoverAttribution()
         configureTag()
+        configureRelatedPosts()
```

Comment:
(Please, feel free to disregard!) If RelatedPosts is a VC that will be displayed via a segue, maybe this setup helper should live within the `prepare(for segue: UIStoryboardSegue` helper?